platform/andes/ae350/trigger.c

```diff
@@ -0,0 +1,287 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2019 Andes Technology Corporation
+ *
+ * Authors:
+ *   Zong Li <zong@andestech.com>
+ *   Nylon Chen <zong@andestech.com>
+ */
+
```

Comment:
I will drop it for now.

@lbmeng  @avpatel 
Will debugging SBI call be supported in the future?

Reply:
In debug spec, the trigger modules can be accessed by m-mode and d-mode, I had pointed out this in previous proposal whether we need to define a SBI ID for it.
So far, there are no relevant SBI interface to use trigger modules in Linux mainline, you may implement it at the same time if needed.
Otherwise, before the debugging SBI calls becomes standard SBI extension, you can use vendor specific SBI calls when opensbi implementation providing SBI 0.2 spec.